code/modules/surgery/organic_steps.dm

```diff
@@ -116,7 +116,7 @@
 	return TRUE
 
 /datum/surgery_step/saw/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results)
-	target.apply_damage(50, BRUTE, "[target_zone]", wound_bonus=CANT_WOUND)
+	target.apply_damage(10, BRUTE, "[target_zone]", wound_bonus=CANT_WOUND)
```

Comment:
Huh. Interesting. Not a big change but still interesting that surgery dealt damage.